<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>677</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1478</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog
{
background:url(&quot;wall4.jpg&quot;);
}
*{
font-family:&quot;Arial&quot;, Gadget, sans-serif;
font-size:16px;

}
#label_3
{
text-align:center;
align-item:center;
background:transparent;
boarder:none;
font-family:Stencil Std, fantasy;
font-weight:bold;
font-size:30px;
}
QFrame
{
   background-color: rgba( 230, 230, 216, 80% );
	border-radius:10px;
	border:1px solid grey;
   
   
}
QPushButton
{
	background:#B0B0B0;
	border-radius:10px;
	
}
QPushButton:Hover
{
	background:red;
	border-radius:10px;
	cursor:pointer;
}
QLabel
{
border:3px solid black
}
#face
{
background:transparent;
border:none;
}
#hand
{
background:transparent;
border:none;
}</string>
  </property>
  <widget class="QLabel" name="processedImgLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>821</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(E:/FYP 2019-20/Code/FaceRecognition-master/Wallpaper/cam)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Wallpaper/cam.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>590</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Wallpaper/icons8-camera-64 (2).png</normaloff>Wallpaper/icons8-camera-64 (2).png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>590</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Wallpaper/icons8-camera-64.png</normaloff>Wallpaper/icons8-camera-64.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="lockBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Unlock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="int_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>590</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Select Intrusion</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Wallpaper/icons8-camera-64.png</normaloff>Wallpaper/icons8-camera-64.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="detailBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>370</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>User Details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>1101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1081</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>                                    DOOR LOCK APPLICATION</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>590</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Wallpaper/icons8-camera-64.png</normaloff>Wallpaper/icons8-camera-64.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
